src/helpers/format.ts

```diff
@@ -42,9 +52,45 @@ export function formatYesNoQueryString(str: string) {
   const formatted = title.substring(0, title.length - 2);
 
   // Return only the first 55 characters
-  if (formatted.length > 55) {
-    return `${formatted.substring(0, 56)}...`;
+  if (formatted.length > maxLength) {
+    return `${formatted.substring(0, maxLength + 1)}...`;
   } else {
     return formatted;
   }
 }
+
+type FormatRequestTitleParams = {
+  identifier?: string;
+  ancillaryData?: string;
+  maxTitleLength?: number;
+  defaultTitle?: string;
+};
```

Comment:
Although not strictly required, I believe TypeScript recommends making objects like this interfaces, not types.

Reply:
oh interesting, ok